requirements/22/index.html

```diff
@@ -218,6 +218,10 @@ <h3>Success Criteria Considerations</h3>
 				<li> Use glossary definitions to simplify and shorten all Success Criteria for shared or ambiguous terms.</li>
 			</ol>
 			</section>
+			<section>
+				<h3>Research</h3>
+				<p>The creation process for success criteria should be data-informed and evidence-based where possible. Research and evidence are influenced by the number of people with a particular disability, by the size of the body of research, and by the difficulty in capturing data regarding some disabilities or combination of disabilities. The intent is to make informed decisions wherever possible to ensure that the needs of all people with disabilities are prioritized, including needs that differ from the majority. In situations where there is no evidence or research, valid data-gathering methods should be used to obtain and evaluate information from advocacy groups, people with lived experience and other subject matter experts.</p>
```

Comment:
I find the phrase "including needs that differ from the majority" a bit confusing. The needs differ from the majority of users? Or is the intent to justify approving ways to address needs without consensus?

How about ending the sentence at "are prioritized."?

Reply:
Also, we need a comma after "lived experience".